cmd/run-plugin/run.go

```diff
@@ -91,7 +91,7 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	ctrlC := make(chan os.Signal, 1)
+	ctrlC := make(chan os.Signal, 4) // We expect up to four ctrl-C inputs.
```

Comment:
Four?  The first does a cancel; the second issues a warning to the user; the third causes the process to exit.  What does the fourth do and how does it do it?

Reply:
Just an extra to prevent deadlocking. I'll update the comment.

Reply:
Not to quibble or anything, but the fourth one will never be received from the channel, since there are only three receive operations coded.  (Obviously, it doesn't really hurt to buffer four...there's just no point to it that I can see.)